pkg/app/piped/cmd/piped/piped.go

```diff
@@ -241,6 +242,11 @@ func (p *piped) run(ctx context.Context, input cli.Input) (runErr error) {
 	}
 	otel.SetTracerProvider(tracerProvider)
 
+	// if control plane's flag monitoring.enabled is false, otel provider logs errors.
+	// it's no problem but we don't want to see it.
+	// so we discard the logger.
+	otel.SetLogger(logr.Discard())
+
```

Comment:
📝 It's ok for now. However, if users want to investigate the otel behavior, it might be necessary. 
If so, it would be nice to add the flag on whether to show the log.